values/harbor/harbor-raw.gotmpl

```diff
@@ -30,3 +33,38 @@ resources:
     issuerRef:
       name: custom-ca
       kind: ClusterIssuer
+{{- if eq $sp.type "minioLocal" }}
```

Comment:
looks like the same secrets will be defined for other apps in different namespaces. It would we a good idea to abstract this part as a reusable template file.

Reply:
to be done later